octoprint_octolapse/render.py

```diff
@@ -569,24 +568,31 @@ def _render(self):
         self._on_complete()
 
     @staticmethod
-    def _get_vcodec_from_extension(extension):
-        default_codec = "mpeg2video"
-
-        if extension in ["mpeg", "vob"]:
-            return "mpeg2video"
-        elif extension in ["mp4", "avi"]:
-            return "mpeg4"
-        elif extension == "flv":
-            return "flv1"
-        elif extension == "gif":
-            return "gif"
-        else:
-            return default_codec
+    def _get_extension_from_output_format(output_format):
```

Comment:
I like what you did here :)

Reply:
This is showing as requested changes D:
---
Everything else is changed as requested though :)